.eslint-dictionary.json

```diff
@@ -1,201 +1,394 @@
 [
+  "accessor",
+  "adaptor",
+  "adaptors",
+  "addgraphqldatasource",
+  "admingroup",
   "adminui",
+  "aggs",
+  "allowlist",
+  "alphanumerics",
   "amazonaws",
+  "amazoncognito",
+  "amd64",
+  "amplifyadmin",
+  "amplifyapp",
+  "amplifyconfig",
+  "amplifyconfiguration",
+  "amplifyconsole",
+  "amplifyhosting",
   "amplifymeta",
+  "amplifyrc",
+  "amplifytools",
+  "ansi",
   "apigateway",
   "apigw",
   "apis",
   "apns",
+  "appleid",
+  "appstate",
   "appsync",
   "archiver",
   "argv",
   "arn",
   "arns",
   "authorizer",
+  "auths",
   "authuser",
+  "autogenerated",
   "awaiter",
   "aws",
+  "awscfn",
   "awscloudformation",
+  "awscognitouserpool",
+  "awsconfig",
+  "awsconfiguration",
   "awsmobile",
   "axios",
   "babelrc",
   "backend",
+  "basedir",
+  "birthdate",
   "bitwise",
   "bool",
+  "browserify",
+  "browserslist",
+  "bugfix",
   "buildable",
   "buildspec",
   "callout",
   "camelcase",
+  "cancellable",
+  "cancelled",
+  "captcha",
   "cfn",
+  "cfnlambda",
+  "chatbot",
+  "chatbots",
+  "chdir",
   "checkbox",
-  "chown",
+  "checkmark",
+  "chokidar",
+  "cicd",
   "cidr",
   "cidrs",
   "circleci",
   "cloudformation",
   "cloudfront",
+  "cloudwatch",
+  "cname",
   "codebase",
   "codegen",
   "codepipeline",
+  "codesuite",
   "cognito",
+  "colocate",
   "columnify",
+  "commitlint",
   "commonjs",
   "cors",
-  "createCipheriv",
+  "createcipheriv",
   "creds",
   "cron",
+  "csproj",
```

Comment:
should this be included?

Reply:
There are template paths that include this. It's a C# project file extension, and is used in a few template paths.

Reply:
This is used in filenames